samples/secure_tunneling/tunnel_notification/README.md

```diff
@@ -21,7 +23,7 @@ On startup, the sample will wait until it receives, and then displays the tunnel
         "iot:Receive"
       ],
       "Resource": [
-        "arn:aws:iot:<b>region</b>:<b>account</b>:topic/$aws/things/<thing_name>/tunnels/notify"
+        "arn:aws:iot:region:account:topic/$aws/things//tunnels/notify"
```

Comment:
Probably use 
`arn:aws:iot:<region>:<account>:topic/$aws/things/<thing_name>/tunnels/notify` to indicate the user should replace the value of region and account?